### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by felsweg**
solves #1

### CODE REVIEW

**Review by wusyong — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contributing Guide
+
+Hello! We, the maintainers, are very pleased that you are interested in contributing to Verso. However, before you submit your contribution, please take a moment to read the [Code of Conduct](CODE_OF_CONDUCT.md) and the appropriate section for the contribution you plan to make:
```
Comment (Mikopet) [inline .github/CONTRIBUTING.md:3]: Instead `Verso`, put `Fortress`.

Comment (Mikopet) [reply .github/CONTRIBUTING.md:3]: link is also broken, no code of conduct file. (that could be copied from verso repo I guess.)

**Review by Mikopet — CHANGES_REQUESTED** [review]

**Review by tensor-programming — APPROVED** [review]

**Comment by felsweg** [comment]
@Mikopet may you kindly approve the changes? thank you

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Mikopet) [inline .github/CONTRIBUTING.md:3]:
```suggestion
Hello! We, the maintainers, are very pleased that you are interested in contributing to Fortress. However, before you submit your contribution, please take a moment to read the [Code of Conduct](CODE_OF_CONDUCT.md) and the appropriate section for the contribution you plan to make:
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Issue Reporting Guidelines
+
+- The issue list on this repo is **exclusively** for bug reports and feature requests. Non-conforming issues will be closed immediately.
+
+- If you have a question, you can get quick answers from the [Verso Zulip](https://versotile.zulipchat.com/).
+
+- Try to search for your issue, it may have already been answered or even fixed in the main branch (`main`).
+
+- Check if the issue is reproducible with the latest version of Verso. If you are using a nightly release, please indicate the specific version you are using.
```
Comment (Mikopet) [inline .github/CONTRIBUTING.md:16]:
```suggestion
- Check if the issue is reproducible with the latest version of Fortress. If you are using a nightly release, please indicate the specific version you are using.
```

**Review by Mikopet — CHANGES_REQUESTED** [review]

**Review by felsweg — COMMENTED** [review]
accept requested changes

**Review by tensor-programming — APPROVED** [review]

**Pull request merged**